releasenotes/notes/value-sort-histogram-1b7033bdfe612f94.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    Two new options, ``'value'`` and ``'value_desc'`` were added to the
+    ``sort`` kwarg of the :func:`qiskit.visualization.plot_histogram` function.
+    When ``sort`` is set to either of these options the output visualization
+    will sort the x axis based on the maximum probability for each bitstring.
```

Comment:
this behavior should probably be in the docstring too (I think it's non-obvious how sort='value' will work when there are multiple histograms being displayed).